SatPiCountingCars.py

```diff
@@ -0,0 +1,214 @@
+# -*- coding: utf-8 -*-
+
```

Comment:
can we also add the shebang indicating the python environment?  like `#! /usr/bin/env python3` 
or i can do it myself too haha